build-index/tagserver/server.go

```diff
@@ -484,29 +470,29 @@ func buildPaginationOptions(u *url.URL) ([]backend.ListOption, error) {
 func buildPaginationResponse(u *url.URL, continuationToken string,
 	result []string) (interface{}, error) {
```

Comment:
Returning interface{} is dangerous and cause runtime issues. Only do it when you absolutely have to and comment on the return types.

Reply:
Initial, buildPaginationResponse was sending different responses based on argument, not anymore. Make sense to not send interface now.